### DISCUSSION TITLE
Sentence Mixing From Audio Files

### DISCUSSION THREAD

**Discussion by insertnameuserhere** [otomad/OtomadHelper#76] [Ideas]
[this](https://github.com/otomad/OtomadHelper/issues/43) but from audio files

**Comment by otomad** [comment]
The feature will rely on a special file that records the time position and pitch of each word in the video, and **of course the audio**.

It just may not support the currently commonly used voicebank formats, such as UTAU, etc. You may need to convert them to the required format. After all, you don't want to import a bunch of scattered single-word audio files into VEGAS.
